### ISSUE TITLE
Add a show/hide password feature

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#1724]
## Description of issue

That might be a good a11y improvement we could make in the future. Here is a gov.uk blog post where they describe how they implemented a "show/hide password" feature: https://technology.blog.gov.uk/2021/04/19/simple-things-are-complicated-making-a-show-password-option/

Describe what is failing
- include what is failing
- include an image
- include context

## Potential fix

- What change could fix this issue?
- What nuggets of information could help us fix the issue

## Resources

- Links to documentation or articles that could be helpful
- examples: WCAG docs, ARIA docis, web.dev, Deque, Webaim...

**Comment by yaelberger-commits** [comment]
we should probably think about having 2 password fields too - its super eassy to create a typo and not know your password!
